phpstan.neon.dist

```diff
@@ -10,7 +10,6 @@ parameters:
 		- plugins/performance-lab/load.php
 	bootstrapFiles:
 		- tools/phpstan/constants.php
-		- plugins/performance-lab/load.php
```

Comment:
It’s unclear to me why this file was included in the `bootstrapFiles`, considering `PERFLAB_PLUGIN_DIR_PATH` is already defined in `tools/phpstan/constants.php`:  
https://github.com/WordPress/performance/blob/6f588eef022354d9f98c950df9146586c5425e66/tools/phpstan/constants.php#L21
After removing the file, the warnings about the duplicate constant definition no longer appear.

Reply:
It was added by @westonruter in https://github.com/WordPress/performance/pull/1658/commits/5509f2cf1735917db8bcfec7bd2a93b6c879d996#diff-4b1024a6caff35a4396a209a68a5d634397320ff3b049b7c59e4937a78bd7d07R21 not sure why 🤔

Reply:
The constant `PERFLAB_PLUGIN_DIR_PATH` was likely added to `tools/phpstan/constants.php` to avoid this error:

```plaintext
 ------ ----------------------------------------------------------------------------- 
  Line   plugins/performance-lab/includes/admin/load.php                              
 ------ ----------------------------------------------------------------------------- 
  232    Parameter #1 $value of function trailingslashit expects string, null given.  
 ------ ----------------------------------------------------------------------------- 


 [ERROR] Found 1 error                                                                                                  
```

Currently, both `constants.php` and `plugins/performance-lab/load.php` define `PERFLAB_PLUGIN_DIR_PATH`. However, the one in `plugins/performance-lab/load.php` evaluates to `null`:
<https://github.com/WordPress/performance/blob/b1bf1ae9ddbbbdf466ae48dcd06ae40f9a6938de/plugins/performance-lab/load.php#L23-L24>

This issue also becomes apparent if the order of files in `bootstrapFiles` is changed as follows:

```diff
diff --git a/phpstan.neon.dist b/phpstan.neon.dist
index c66cc423..9c617d12 100644
--- a/phpstan.neon.dist
+++ b/phpstan.neon.dist
@@ -9,9 +9,9 @@ parameters:
 		- performance.php
 		- plugins/performance-lab/load.php
 	bootstrapFiles:
-		- tools/phpstan/constants.php
 		- plugins/performance-lab/load.php
+		- tools/phpstan/constants.php
 		- plugins/webp-uploads/load.php
 	scanDirectories:
 		- vendor/wp-phpunit/wp-phpunit/
 	scanFiles:

```

With this change, both the warning and the error occur:

```plaintext
PHP Warning:  Constant PERFLAB_PLUGIN_DIR_PATH already defined in /home/shyam/Workspace/rtCamp/wp-performance/tools/phpstan/constants.php on line 21
Warning: Constant PERFLAB_PLUGIN_DIR_PATH already defined in /home/shyam/Workspace/rtCamp/wp-performance/tools/phpstan/constants.php on line 21
 206/206 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ------ ----------------------------------------------------------------------------- 
  Line   plugins/performance-lab/includes/admin/load.php                              
 ------ ----------------------------------------------------------------------------- 
  232    Parameter #1 $value of function trailingslashit expects string, null given.  
 ------ ----------------------------------------------------------------------------- 

 [ERROR] Found 1 error                                                                                                  
```

Since none of the other constants defined in `plugins/performance-lab/load.php` appear to be required for static analysis, removing this file from `bootstrapFiles` seems appropriate.